### DISCUSSION TITLE
After reboot and restart 20 out of 59 plots disappeared .

### DISCUSSION THREAD

**Discussion by RollingRoland** [Chia-Network/chia-blockchain#3483] [Support]
I just rebooted my PC for some maintenance. After I restarted Chi only 39 out 59 plots I originally had appeared farming. The files are all still there. And I tried all the adding plot directories, delete and re add and refresh of the plots. I tested in one of the smaller directories where I had 5 plots. I was able to sort them 3 will show and 2 will not.

For any help figuring this out I would be thankful.

**Comment by RollingRoland** [comment]
In the release notes for 1.1.3 version they have a line that says:
"Some duplicate plots were not being caught and logged by harvester on loading."
Is it possible that this "improvement" is responsible for this and is falsely flagging 20 of my plots as duplicates?
Edit:
After I rolled back to version 1.1.2. But the problem still persists. So it might be something else.

**Reply by sargonas** [reply]
If you want to confirm for sure whether or not duplicate plots are part of the problem, on the command line you can run `chia plots check -l -n 0`

This will at least help rule that out, before you have to go too far down the rabbit hole.

**Reply by sargonas** [reply]
While I’m at it, if you dig through the documentation in the wiki for the command line interface, there’s other plot check flags you can run that may help you highlight if any of those plots have issues that’s blocking them

**Comment by Szarlak** [comment]
Hello
Got exactly the same magic, some of my plots disappear from list, however .plot file is still on disc. None of the commands in the wiki (check) helps me (they cant see the plot) or i do something wrong ^^. These plots was farming status before. Anyone defeat this situaton?

**Reply by oytp** [reply]
While pool miner is running, some of your plots couldn't be shown. Try: reboot OS -> launch CHIA client. And only then start pool client if you use it

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
